matter/wifi/wfx_host_events.h

```diff
@@ -76,95 +76,108 @@
-#define STRUCT_PBUF	(struct pbuf *)0
-#define INT_VALUE_64	  64
-#define INT_VALUE_10	  10
-#define	PRIORITY_0		  0
-#define HEX_VALUE_FF	  0XFF
-#define SHIFT_8		      8
-#define SHIFT_16  	    16
-#define SHIFT_24		    24
-
-#define DELAY_10MS        10
-#define DELAY_3MS         3
-#define UINT8_BUFF_0  (uint8_t *)0
-#define MAX_XLEN          16
-#define MIN_XLEN          0
-#define VOID_BUFF0     (void *)0
-#define PINOUT_CLEAR      0
-#define PINOUT_SET        1
-#define NVIC_PRIORITY     5
-#define CB_USER_PARAM  (void *)0
-#define CB_VALUE        (DMADRV_Callback_t)0
-
-#define DELAY0	        0
-#define HANDLE0         (TimerHandle_t)0
-#define TIMER_ID0       (void *)0
-#define BLOCK_TIME      0
-#define CONVERSION_VALUE 	1000
-
-#define JOIN_CNT0 	        0
-#define BUFF_SIZE_28	      28
-#define BUF_SIZE_128	      128
-#define RSI_DRIVER_STATUS   0
-#define OPER_MODE_0         0
-#define COEX_MODE_0         0
-#define RESP_BUFF_SIZE      6
-#define AP_CHANNEL_NO_0     0
-#define SCAN_BITMAP_OPTN_1  1
-#define JOIN_RETRIES_CNT_5  5
-#define IP_CONF_RSP_BUFF_LENGTH_4     4
-#define STATION             0
-#define BG_SCAN_RES_SIZE    500
-
-#define SPI_CONFIG_SUCESS	 0
+#define	PROBE_NUM_REQ_1			1
+
+#define PINOUT_CLEAR_STATUS    		0
+#define TICKS_TO_WAIT_0   		0
+#define TICKS_TO_WAIT_3   		3
+#define TICKS_TO_WAIT_1   		1
+#define TICKS_TO_WAIT_500   		500
+
+
+// TASK and Interrupt Macros
+#define CLEAR_BUFFER			0
+#define SUCCESS_STATUS			1
+#define LINK_UP			    	1
+#define LINK_DOWN			0
+#define MAC_48_BIT_SET			1
+#define LEN_0		        	0
+#define LEN_60		      		60
+#define STRUCT_PBUF			(struct pbuf *)0
+#define INT_VALUE_64	  		64
+#define INT_VALUE_10	  		10
+#define	PRIORITY_0		  	0
+#define HEX_VALUE_FF	  		0XFF
+#define SHIFT_8		      		8
+#define SHIFT_16  	    		16
+#define SHIFT_24		    	24
+
+// Timer Delay
+#define DELAY_10MS        		10
+#define DELAY_3MS         		3
+#define MAX_XLEN          		16
+#define MIN_XLEN          		0
+#define PINOUT_CLEAR      		0
+#define PINOUT_SET        		1
+#define WFX_SPI_NVIC_PRIORITY     	5
+#define WFX_GPIO_NVIC_PRIORITY     	5
+#define CB_VALUE        		(DMADRV_Callback_t)0
+
+#define DELAY0	        		0
+#define HANDLE0         		(TimerHandle_t)0
+
+/* TIMER_TICKS_TO_WAIT Specifies the time, in ticks, that the calling task should
+ * be held in the Blocked state to wait for the start command to be successfully
+ * sent to the timer command queue.
+ */
+#define TIMER_TICKS_TO_WAIT_0      	0
+
+#define CONVERT_SEC_TO_MSEC 		1000
+#define CONVERT_USEC_TO_MSEC 		(1/1000)
+
+#define RSI_RESPONSE_MAX_SIZE		28
+#define RSI_RESPONSE_HOLD_BUFF_SIZE	128
+#define RSI_DRIVER_STATUS   		0
+#define OPER_MODE_0         		0
+#define COEX_MODE_0         		0
+#define RESP_BUFF_SIZE      		6
+#define AP_CHANNEL_NO_0     		0
+#define SCAN_BITMAP_OPTN_1  		1
+#define IP_CONF_RSP_BUFF_LENGTH_4     	4
+#define STATION             		0
+#define BG_SCAN_RES_SIZE    		500
+
+#define SPI_CONFIG_SUCESS	 	0
```

Comment:
Can you review the list of defines and make sure all the ones present are used and necessary.

Reply:
reviewed and removed unnecessary macros